ci/common/test-atacworks.sh

```diff
@@ -20,3 +20,6 @@ if [ "${TEST_ON_GPU}" == '1' ]; then
 else
     python -m pytest -m cpu tests/ -v
 fi
+
+# Install a wheel package
```

Comment:
@ntadimeti `create` instead of `install`

Also, this command should be executed when running a CPU job, not a GPU job. We should execute `test-atacworks.sh` twice in the CPU job- for the build from source code and after installing the `.whl` file we created.

Reply:
Done

Reply:
done.